api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -507,6 +507,42 @@ message Cluster {
       google.protobuf.UInt32Value hash_balance_factor = 2 [(validate.rules).uint32 = {gte: 100}];
     }
 
+    // Configuration for :ref:`slow start mode <arch_overview_load_balancing_slow_start>`.
+    message SlowStartConfig {
+      // Endpoint warming policy defines additional condition (apart from condition that endpoint creation duration should be within slow start window)
+      // for endpoint to enter slow start mode.
+      // Refer to :ref:`endpoint warming policy types <arch_overview_load_balancing_slow_start_endpoint_warming_policy_types>`
+      // for an explanation on each policy type.
+      enum EndpointWarmingPolicy {
+        // Endpoint would enter slow start immediately.
+        NO_WAIT = 0;
+
+        // Endpoint would enter slow start upon passing first active healthcheck.
+        WAIT_FOR_FIRST_PASSING_HC = 1;
+      }
+
+      // Size of slow start window in seconds, defaults to 0.
+      // If set to value greater than 0, the newly created host remains in slow start mode starting from it's creation time
+      // for the duration of slow start window.
+      uint32 slow_start_window = 1 [(validate.rules).uint32 = {gte: 0}];
```

Comment:
validation not needed, though what would be the use case for setting this to 0?

Reply:
this would effectively disable slow start. Right, uint32 cannot be negative :)

Reply:
My point is >= is guaranteed, so I would either make it > or remove it. I think you want > 0? The text above about defaulting to 0 doesn't make sense I don't think. I think the lack of a SlowStartConfig at all is enough to enable/disable slow start, so for this param I would just make it > 0 and update the text.